### PR TITLE
wrapGAppsHook: add wrapGApp helper

### PR DESCRIPTION
**Pull request by jtojnar**
For parity with Qt.

### CODE REVIEW

**Comment by worldofpeace** [comment]
Can we have dde-daemon use this?
https://github.com/NixOS/nixpkgs/blob/master/pkgs/desktops/deepin/dde-daemon/default.nix#L118

**Comment by jtojnar** [comment]
Hmm, for some reason I did not push the rest.

File: pkgs/tools/graphics/zbar/default.nix [hunk 1]
```diff
@@ -68,7 +68,7 @@ stdenv.mkDerivation rec {
   dontWrapGApps = true;
 
   postFixup = lib.optionalString enableVideo ''
-    wrapProgram "$out/bin/zbarcam-gtk" "''${gappsWrapperArgs[@]}"
+    wrapGApp "$out/bin/zbarcam-gtk"
```
Comment (worldofpeace) [inline pkgs/tools/graphics/zbar/default.nix]: wrapGAppsHook isn't in nativeBuildInputs

Comment (jtojnar) [reply pkgs/tools/graphics/zbar/default.nix]: Bah, missed that when rebasing #68389

**Pull request merged**